### PR TITLE
VMHostGraphics DSC Resource and VMHostGraphicsDevice DSC Resource

### PR DESCRIPTION
**Pull request by SimeonGerginov**
### Added
- Added VMHostGraphics DSC Resource.
- Added Unit Tests for VMHostGraphics DSC Resource.
- Added Integration Tests for VMHostGraphics DSC Resource.
- Added Documentation and example Configuration for VMHostGraphics DSC Resource.
- Added VMHostGraphicsDevice DSC Resource.
- Added Unit Tests for VMHostGraphicsDevice DSC Resource.
- Added Integration Tests for VMHostGraphicsDevice DSC Resource.
- Added Documentation and example Configuration for VMHostGraphicsDevice DSC Resource.

### CODE REVIEW

**Comment by SimeonGerginov** [comment]
Example usage of VMHostGraphics DSC Resource:

```
VMHostGraphics VMHostGraphics {
    Name = $Name
    Server = $Server
    Credential = $Credential
    DefaultGraphicsType = 'Shared'
    SharedPassthruAssignmentPolicy = 'Performance'
    DeviceId = '0000:00:00.0'
    DeviceGraphicsType = 'SharedDirect'
    RestartTimeout = 10
}
```

File: Documentation/DSCResources/VMHostGraphics/VMHostGraphics.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+| **SharedPassthruAssignmentPolicy** | Mandatory | SharedPassthruAssignmentPolicy | The policy for assigning shared passthrough VMs to a host graphics device. | Performance, Consolidation |
+| **DeviceId** | Optional | string | The Graphics device identifier (ex. PCI ID). ||
+| **DeviceGraphicsType** | Optional | GraphicsType | The graphics type for the specified Device in 'DeviceId' property. | Shared, SharedDirect, Unset |
+| **RestartTimeout** | Optional | int | The time in minutes to wait for the VMHost to restart before timing out and aborting the operation. The default value is 5 minutes. ||
+
+## Prerequisite
+The specified VMHost must be in **Maintenance** mode.
+
+## Description
+The resource is used to update the Graphics configuration by changing the DefaultGraphicsType and SharedPassthruAssignmentPolicy values. If a Graphics Device is specified it also changes its Graphics Type value. The resource also restarts the VMHost after a successful Update operation.
```
Comment (dmilov) [inline Documentation/DSCResources/VMHostGraphics/VMHostGraphics.md]: As we discussed split to two resources one for Host configuration and another for Device configuration

File: Source/VMware.vSphereDSC/DSCResources/VMHostGraphics.ps1 [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+[DscResource()]
+class VMHostGraphics : VMHostBaseDSC {
+    <#
+    .DESCRIPTION
+
+    Specifies the default graphics type for the specified VMHost. This default value is overridden if an individual device graphics type is specified.
+    If the specified VMHost supports a single graphics type, specifying an individual graphics device is optional.
+    #>
+    [DscProperty(Mandatory)]
+    [GraphicsType] $DefaultGraphicsType
```
Comment (dmilov) [inline Source/VMware.vSphereDSC/DSCResources/VMHostGraphics.ps1]: This one would be just GraphicsType for the VMHost resource

File: Documentation/DSCResources/VMHostGraphics/VMHostGraphics.md (hunk 1 above)
Comment (SimeonGerginov) [reply Documentation/DSCResources/VMHostGraphics/VMHostGraphics.md]: Done.

File: Source/VMware.vSphereDSC/DSCResources/VMHostGraphics.ps1 (hunk 2 above)
Comment (SimeonGerginov) [reply Source/VMware.vSphereDSC/DSCResources/VMHostGraphics.ps1]: Done.

File: Documentation/DSCResources/VMHostGraphicsDevice/VMHostGraphicsDevice.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## Prerequisite
+The specified VMHost must be in **Maintenance** mode.
+
+## Description
+The resource is used to update the Graphics Type of the specified Graphics Device. The resource also restarts the VMHost after a successful Update operation.
+
+## Examples
+
+### Example q
```
Comment (dmilov) [inline Documentation/DSCResources/VMHostGraphicsDevice/VMHostGraphicsDevice.md]: fix

Comment (SimeonGerginov) [reply Documentation/DSCResources/VMHostGraphicsDevice/VMHostGraphicsDevice.md]: Done.

**Pull request merged**